mmv1/products/datafusion/api.yaml

```diff
@@ -88,6 +88,7 @@ objects:
         values:
           - :BASIC
           - :ENTERPRISE
+          - :DEVELOPER
```

Comment:
Can we add a description of the enum above?

Reply:
done